plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/editor/ERDEditorPart.java

```diff
@@ -116,7 +119,7 @@
  */
 public abstract class ERDEditorPart extends GraphicalEditorWithFlyoutPalette
         implements DBPDataSourceTask, IDatabaseModellerEditor, ISearchContextProvider, IRefreshablePart, INavigatorModelView {
-    private static final Log log = Log.getLog(Searcher.class);
+    private static final Log searcherLog = Log.getLog(Searcher.class);
```

Comment:
Why this change?

Reply:
Because it's log for Searcher class which is placed in the parent class. I assume it would be better to highlight it to prevent accidentally usage in parent class.

Reply:
I see. Is it possible to move it into the `Searcher` class?

Reply:
Unfortunately, it's not a thing until Java 16. That's sad.

Reply:
It's not possible due to java language level.

Reply:
static declarations in inner class are not supported yet